x/axelarnet/client/cli/query.go

```diff
@@ -79,3 +88,49 @@ func getParams() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// getCmdChains returns the query to get all Cosmos chains
+func getCmdChains() *cobra.Command {
```

Comment:
I think having a CLI wrapper would be convenient. But I'm open to not adding it, and deprecating the evm chains if you guys prefer